deploy/docker-ephemeral/build/README.md

```diff
@@ -0,0 +1,29 @@
+A makefile that uses docker.io, and qemu-user-static to build dependencies for our integration tests.
+
+Builds docker images for multiple architectures. allows for -j to build multiple images at once.
+
+# Setup
+
+## Docker
+docker.io is only available from debian testing or unstable.
+`apt-get install docker.io`
+
+after installing docker-io, add the user you are running as to the docker group, and restart X.
```

Comment:
I would just link to this section instead: https://github.com/wireapp/wire-server/blob/develop/doc/Dependencies.md#docker (and update that part, if needed)

Reply:
done.